<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabEstoque</class>
 <widget class="QWidget" name="TabEstoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabEstoques">
      <attribute name="title">
       <string>Estoques</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="WidgetEstoques" name="widgetEstoques" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProdutos">
      <attribute name="title">
       <string>Produtos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="WidgetEstoqueProduto" name="widgetProdutos" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetEstoques</class>
   <extends>QWidget</extends>
   <header>src/widgetestoques.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetEstoqueProduto</class>
   <extends>QWidget</extends>
   <header>src/widgetestoqueproduto.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
